### ISSUE TITLE
Audio scrubbing

### ISSUE THREAD

**Issue by dreammu** [mifi/lossless-cut#1840]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

Like the audio scrubbing feature in  Davinci Resolve, to hear the audio frame by frame.

**Comment by mifi** [comment]
duplicate of #20

tldr: i don't know how to implement that easily with html5 video
